styles/style.css

```diff
@@ -0,0 +1,526 @@
+/******************/
+/* GENERAL STYLES */
+/******************/
+
+:root {
+    --yellow: rgb(225, 168, 25);
+    --pink: rgb(237, 106, 90);
+    --dark-blue: rgb(40, 48, 68);
+    --light-blue: rgb(120, 161, 187);
+    --mint: rgb(235, 245, 238);
+}
+
+html {
+    font-size: 18px;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+*::selection {
+    background-color: var(--pink);
+    color: white;
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+    font-weight: 500;
+    transition-property: color;
+    transition-duration: .08s;
+    display: inline-block;
+}
+
+a:hover {
+    color: var(--mint);
+}
+
+p a:after, ul a:after {
+    display: block;
+    content: '';
+    border-bottom: 1px solid var(--pink);
+    transform: scaleX(0);
+    transition: transform 250ms ease-in-out;
+}
+
+p a:hover::after, ul a:hover::after {
+    transform: scaleX(1);
+}
+
+hr {
+    background-color: var(--yellow);
+    border: none;
+    height: 1px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+body {
+    background-color: var(--mint);
+    margin: 0;
+    font-family: 'Rubik', sans-serif;
+    color: var(--dark-blue)
+}
+
+body > * {
+    padding: 0 10%;
+}
+
+nav {
+    height: 50px;
+    background-color: var(--pink);
+    width: 100%;
+    padding: 0;
+    position: fixed;
+    top: 0;
+    z-index: 1;
+    border-bottom: .5px solid white;
+    box-shadow: 0px 1px 2px rgb(237, 106, 90);
+}
+
+nav ul {
+    height: 100%;
+    list-style: none;
+    padding: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    margin: 0;
+}
+
+nav a {
+    color: var(--mint);
+    text-transform: uppercase;
+    text-decoration: none;
+    border: none;
+    font-weight: bold;
+}
+
+nav a:hover {
+    color: var(--dark-blue);
+    border-bottom: 1px solid var(--dark-blue);
+}
+
+header {
+    padding-top: 100px;
+}
+
+h1:not(#name), #h1 {
+    text-transform: lowercase;
+    border: 3px solid var(--dark-blue);
+    padding: 5px 10px;
+    width: fit-content;
+    margin: 30px auto 50px auto;
+    text-align: center;
+}
+
+#h1 {
+    background-color: var(--dark-blue);
+    color: var(--mint)
+}
+
+h1:hover:not(#name) {
+    background-color: var(--dark-blue);
+    color: var(--mint)
+}
+
+#name {
+    margin: 0;
+}
+
+p {
+    line-height: 160%;
+}
+
+footer {
+    margin-top: 50px;
+    margin-bottom: 20px;
+    text-align: center;
+    font-size: .8em;
+}
+
+footer > * {
+    margin: 0;
+}
+
+footer p:last-of-type {
+    margin-bottom: 5px;
+    display: inline;
+}
+
+footer ul {
+    list-style-type: none;
+    padding: 0;
+    margin-top: 3px;
+}
+
+footer ul li {
+    display: inline;
+    padding-right: 10px;
+}
+
+footer ul a:hover {
+    color: var(--pink)
+}
+
+.links a {
+    padding: 0 5px;
+    font-size: 30px;
+    border: none;
+    color: var(--yellow);
+    margin-top: 15px;
+}
+
+.links a:hover {
+    color: var(--pink)
+}
+
+/*************************/
+/* HOMEPAGE STYLES */
+/*************************/
```

Comment:
Consider separating your CSS into separate files. People have differing opinions on what's best; try both and see what you prefer!